src/test/kotlin/graphics/scenery/tests/examples/volumes/TransferFunctionEditorExample.kt

```diff
@@ -0,0 +1,161 @@
+
+import bdv.spimdata.XmlIoSpimDataMinimal
+import graphics.scenery.*
+import graphics.scenery.backends.Renderer
+import graphics.scenery.UI.SwingBridgeFrame
+import graphics.scenery.UI.SwingUiNode
+import graphics.scenery.utils.extensions.plus
+import graphics.scenery.utils.extensions.times
+import graphics.scenery.volumes.Colormap
+import graphics.scenery.volumes.TransferFunctionUI
+import graphics.scenery.volumes.Volume
+import org.joml.Vector3f
+import org.scijava.ui.behaviour.ClickBehaviour
+import org.scijava.ui.behaviour.DragBehaviour
+import tpietzsch.example2.VolumeViewerOptions
+import java.io.File
+import java.nio.file.Paths
+
+
+/**
+ * @author Konrad Michel
+ * Example for scenery - swing bridge
+ *
+ * A TransferFunctionEditor example to add, manipulate and remove control points of a volume's transfer function.
+ * Further more able to generate a histogram representation of the volume data distribution to help with the transfer function setup.
+ *
+ * Usage: To enable the UI on the plane click once (Key '1') while hovering over the plane. Key '1' used as normal Mouse-interactions (Clicking and dragging).
+ * Control Points can be dragged, added and removed. A remove happens via Ctrl-Clicking (In this example managed by using Key '2'.
+ */
+class TransferFunctionEditorExample : SceneryBase("TransferFunctionEditor Example", 1280, 720, false) {
+    var maxCacheSize = 512
+    val cam: Camera = DetachedHeadCamera()
+
+    /**
+     * Sets up the example, containing 2 light sources (PointLight), a perspective camera and a volume.
+     * Also adds a SwingUINode containing a SwingBridgeFrame contained by a TransferFunctionUI to manipulate the Volume
+     */
+    override fun init() {
+
+        renderer = hub.add(
+            SceneryElement.Renderer,
+            Renderer.createRenderer(hub, applicationName, scene, windowWidth, windowHeight)
+        )
+
+        val light = PointLight(radius = 15.0f)
+        light.spatial().position = Vector3f(2.0f, 0.0f, 2.0f)*2f
+        light.intensity = 15.0f
+        light.emissionColor = Vector3f(1.0f, 1.0f, 1.0f)
+        scene.addChild(light)
+
+        val light2 = PointLight(radius = 15.0f)
+        light2.spatial().position = Vector3f(-2.0f, 0.0f, -2.0f)*2f
+        light2.intensity = 15.0f
+        light2.emissionColor = Vector3f(1.0f, 1.0f, 1.0f)
+        scene.addChild(light2)
+
+        with(cam) {
+            spatial {
+                position = Vector3f(0.0f, 0.0f, 2.0f)
+            }
+            nearPlaneDistance = 0.01f
+            perspectiveCamera(50.0f, windowWidth, windowHeight)
+
+            scene.addChild(this)
+        }
+
+        val workingDirectoryPath = File("").absolutePath
+
+        val options = VolumeViewerOptions().maxCacheSizeInMB(maxCacheSize)
+        val name = "t1-head"
+        //Currently only .xml volume formats are usable
+        val v = Volume.fromSpimData(XmlIoSpimDataMinimal().load(getDemoFilesPath() + "$name.xml"), hub, options)
```

Comment:
This has four issues:
* the path to the file is missing a path separator
* workingDirectoryPath is unnecessary
* fromSpimData(Xml...) is also unnecessary, please use Volume.fromXML
* name could be inlined, the variable is unnecessary